### ISSUE TITLE
为啥

### ISSUE THREAD

**Issue by shanqiu24** [DuZzzs/monodleX#8]
为什么的运行官方的monodle：
(monodle) G:\WWProject\monodle\experiments\example>python ../../tools/train_val.py --config kitti_example.yaml
2022-10-11 10:46:15,326   INFO  ###################  Training  ##################
2022-10-11 10:46:15,326   INFO  Batch Size: 16
2022-10-11 10:46:15,326   INFO  Learning Rate: 0.001250
然后居然需要过半个小时才出现的训练的epoch进度条，我将原monodle改为单卡运行了，我的是单卡3090
epochs:   0%|                                                                                                                                                                         | 0/140 [00:00<?, ?it/s]
iters:   0%|                                                                                                                                                                          | 0/232 [00:00<?, ?it/s]

**Comment by DuZzzs** [comment]
抱歉，不清楚这个的原因。

**Comment by shanqiu24** [comment]
> 抱歉，不清楚这个的原因。

我用全部的train和val数据训练，然后在test上测试，得到的moderate Car的精度为14.01，比官方论文中报道的12.26高不少，真奇怪，官方的repository里面报道的是13.6、13.7左右，我觉得这才是应该是正常的精度，看了下您跑的原始monodle精度是13.7197，和官方的epository里面报道的差不多。但是我试了下用您的monodleX（加入了对截断obj的处理），主干网络我用的是DLA34，精度居然掉到了13.28，不能理解，照理说对截断物体的处理不会影响模型对正常物体的检测呀😭

**Comment by yaowangyekong** [comment]
> 然后居然需要过半个小时才出现的训练的epoch进度条，我将原monodle改为单卡运行了，我的是单卡3090

你好！请教一下，如何将多卡运行改为单卡运行？代码应该改那一部分呢？

**Comment by shanqiu24** [comment]
> > 然后居然需要过半个小时才出现的训练的epoch进度条，我将原monodle改为单卡运行了，我的是单卡3090
> 
> 你好！请教一下，如何将多卡运行改为单卡运行？代码应该改那一部分

> > 然后居然需要过半个小时才出现的训练的epoch进度条，我将原monodle改为单卡运行了，我的是单卡3090
> 
> 你好！请教一下，如何将多卡运行改为单卡运行？代码应该改那一部分呢？

你直接clone下来跑通就是单卡运行的，不需要做什么改动

**Comment by DuZzzs** [comment]
> > 抱歉，不清楚这个的原因。
> 
> 我用全部的train和val数据训练，然后在test上测试，得到的moderate Car的精度为14.01，比官方论文中报道的12.26高不少，真奇怪，官方的repository里面报道的是13.6、13.7左右，我觉得这才是应该是正常的精度，看了下您跑的原始monodle精度是13.7197，和官方的epository里面报道的差不多。但是我试了下用您的monodleX（加入了对截断obj的处理），主干网络我用的是DLA34，精度居然掉到了13.28，不能理解，照理说对截断物体的处理不会影响模型对正常物体的检测呀😭 

@shanqiu24 可以尝试把截断率调整下，降低截断目标的样本数看下指标是否有提升。再把结果和这个效果差的可视化观察下。
